pom.xml

```diff
@@ -1997,6 +1998,11 @@
                 <artifactId>awaitility</artifactId>
                 <version>${awaitility.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.fabric8</groupId>
+                <artifactId>kubernetes-server-mock</artifactId>
+                <version>${kubernetes-server-mock.version}</version>
+            </dependency>
```

Comment:
Is there a reason why this isn't hazelcast module dependency, but root dependency?

Reply:
this is in the `dependencyManagement` section to control dependency version, the actual dependency is in `hazelcast/pom.xml`